src/main/java/com/microsoft/azure/functions/HttpStatus.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.functions;
+
+/**
+ * Enum to represent HTTP Status codes.
+ * 
+ * @since 1.0
+ * @author Bruno Borges
+ */
+public enum HttpStatus {
+
+    CONTINUE(100), SWITCHING_PROTOCOLS(101), PROCESSING(102), CHECKPOINT(103), OK(200), CREATED(201), ACCEPTED(202),
```

Comment:
*Minor Suggestion* Can we add empty lines to separate 1xx, 2xx, 3xx, 4xx and 5xx to make it clearer?

Reply:
Sure, I will organize using comment.